packages/@sanity/core/src/actions/graphql/listApisAction.js

```diff
@@ -0,0 +1,46 @@
+module.exports = async function listApisAction(args, context) {
+  const {apiClient, output, chalk} = context
+
+  const client = apiClient({
+    requireUser: true,
+    requireProject: true
+  })
+
+  let endpoints
+  try {
+    endpoints = await client.request({
+      url: `/apis/graphql`,
+      method: 'GET'
+    })
+  } catch (err) {
+    if (err.statusCode === 404) {
+      endpoints = []
+    } else {
+      throw err
+    }
+  }
+
+  endpoints = [{
```

Comment:
Forgot to remove the mock endpoints again?

Reply:
🤦‍♀
I guess that's what you get when you don't have tests on this. I'll fix this right away!